frontend/public/learn/Interactive-UI-Elements.elm

```diff
@@ -85,36 +92,38 @@ This is a very high level view of how things work, but it illustrates the
 typical structure of interactive UI elements in Elm. Let's look at the API
 in more detail.
 
-## Inputs
+## Channels
 
-The first thing we do when making an interactive UI element is to create an input:
+The first thing we do when making an interactive UI element is to create a channel:
 
 ```haskell
-type Input a = { signal : Signal a, handle : Handle a }
+type Channel a = Channel
 ```
 
-An `Input` has two distinct parts. One is a `signal` of events coming from UI
+A `Channel` has two distinct uses. One is to provide a `Signal` of events coming from UI
 elements. In our synced checkbox example, these are boolean values indicating
-whether the box should be checked or not. The more subtle part of an `Input` is
-the `handle`. All interactive UI elements will latch on to a handle and report
-their events to the corresponding `Input`. The handle lets us get events from
+whether the box should be checked or not. This `Signal` can be subscribed to. The more subtle part of a `Channel` is
+to provide a `Signal.Message` handler. All interactive UI elements will latch on to a message handler and so report
+their events to the corresponding `Channel`. The message handler lets us get events from
 the UI back into our program without any messy callbacks or event-listeners.
 
-You create an `Input` with the `input` function, like this:
+You create a `Channel` with the `channel` function, like this:
 
 ```haskell
-input : a -> Input a
+channel : a -> Channel a
 
-check : Input Bool
-check = input False
+check : Signal.Channel Bool
+check = Signal.channel False
 ```
 
-The argument to `input` serves as the default value of the input&rsquo;s
-`signal`. So when we create the `check` input we get:
+The argument to `channel` serves as the default value of the channel&rsquo;s
+`Signal`. So when we create the `check` channel we can:
 
-  1. A signal called `check.signal` with the initial value `False`.
-  2. A handle called `check.handle`. Any UI element can refer to `check.handle`
-     which would send events to `check.signal`.
+  1. Subscribe to the channel using `Signal.subscribe check`, to receive all its
+     messages as a `Signal`, the initial value being `False` in the example.
+  2. Send a `Signal.Message` to the `check` channel, by calling `Signal.send check value`.
+     Any UI element can send a message to the `check` channel, which would pass it
+     along through a `Signal` to its subscribers.
```

Comment:
You may wish to clarify that `Signal.send` doesn't send a message but rather just creates it. The sending is handled by the language runtime.